java/src/jmri/jmrit/consisttool/ConsistListCellRenderer.java

```diff
@@ -0,0 +1,90 @@
+package jmri.jmrit.consisttool;
+
+import java.awt.*;
+import java.awt.image.BufferedImage;
+
+import javax.swing.*;
+
+import jmri.*;
+import jmri.jmrit.roster.*;
+import jmri.util.gui.GuiLafPreferencesManager;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ * @author Lionel Jeanson Copyright (c) 2023
```

Comment:
The copyright notice is usually below the description of the class.

Reply:
ok, done, will be in next push.